### DISCUSSION TITLE
Alternative to OpenCollective

### DISCUSSION THREAD

**Discussion by brechtcs** [beakerbrowser/beaker#1658] [General] [answered]
I was wondering if there's another way to sponsor the project. OpenCollective only works with a credit card, but I don't have a one anymore at the moment. Maybe there's an alternative that supports some more payment options?

**Comment by pfrazee** [comment, marked as answer]
I appreciate you asking! There's not, currently. We've been thinking about setting up free-tier/paid-tier services (replacements to Hashbase) to support the browser.
